dev-tools/release/smoketest_rc.sh

```diff
@@ -0,0 +1,220 @@
+}
+trap failMsg1 ERR
+
+
+echo ""
+echo "============================================================"
+echo "Cleaning workspace directory..."
+echo "============================================================"
+set -x
+rm -rf $WORKSPACE
+mkdir -p $WORKSPACE
+cd $WORKSPACE
+set +x
+
+
+echo ""
+echo "============================================================"
+echo "Cloning repositories at the RC tag..."
+echo "============================================================"
+set -x
+mkdir src_clones
+cd src_clones
+git clone --depth=1 --branch=rel/v${FULL_VERSION} git@github.com:apache/geode.git apache-geode-${VERSION}-src
+git clone --depth=1 --branch=rel/v${FULL_VERSION} git@github.com:apache/geode-examples.git apache-geode-examples-${VERSION}-src
+git clone --depth=1 --branch=rel/v${FULL_VERSION} git@github.com:apache/geode-native.git apache-geode-native-${VERSION}-src
+git clone --depth=1 --branch=rel/v${FULL_VERSION} git@github.com:apache/geode-benchmarks.git apache-geode-benchmarks-${VERSION}-src
+cd $WORKSPACE
+set +x
+
+
+echo ""
+echo "============================================================"
+echo "Downloading artifacts"
+echo "============================================================"
+
+DOWNLOAD_URL="https://dist.apache.org/repos/dist/dev/geode/$FULL_VERSION"
+set -x
+wget $DOWNLOAD_URL -m -np -nv
+set +x
+
+DOWNLOAD_DIR=$WORKSPACE/dist.apache.org/repos/dist/dev/geode/$FULL_VERSION
+echo ""
+echo "============================================================"
+echo "Untarring source releases"
+echo "============================================================"
+set -x
+mkdir -p src_releases
+cd src_releases
+ls $DOWNLOAD_DIR/*src.tgz | xargs -n1 -I {} bash -c "tar xzf {}"
+cd -
+set +x
+
+echo ""
+echo "============================================================"
+echo "Verifying Signatures"
+echo "============================================================"
+
+set -x
+cd $DOWNLOAD_DIR
+ls *.zip *.tar.gz *.tgz | xargs -n1 -I {} bash -c "shasum -c {}.sha*"
+ls *.zip *.tar.gz *.tgz | xargs -n1 -I {} bash -c "gpg --verify {}.asc"
+cd $WORKSPACE
+set +x
+
+echo ""
+echo "============================================================"
+echo "Comparing Source releases with the git tags"
+echo "============================================================"
+set -x
+diff -r -q \
+    -x build \
+    -x gradlew \
+    -x gradle-wrapper.jar \
+    -x gradlew.bat \
+    -x .git \
+    -x .gitignore \
+    -x .buildinfo \
+    -x .gradle \
+    -x .travis.yml \
+    -x jpf.properties \
+    -x .gitattributes \
+    -x geode-old-versions \
+    -x .asf.yaml \
+    -x KEYS \
+    $WORKSPACE/src_clones $WORKSPACE/src_releases
+    
+set +x
+
+echo ""
+echo "============================================================"
+echo "Building from source releases"
+echo "============================================================"
+set -x
+
+cd $WORKSPACE/src_releases/apache-geode-${VERSION}-src
+./gradlew build -Pversion=${VERSION}
```

Comment:
is `-x test` intentionally omitted from this one?  since minimum ASF validation just requires compile, maybe running tests too should be an optional flag to the script?

Reply:
Intentionally, yes. Is it a good idea, I'm not sure. test for the geode project just runs the unit tests, which is kinda nice. The benchmarks and examples have much long suites that get run as part of test. But maybe we should still run them? I'm not sure. For now I left it the way it is but I'm open to whatever we think makes sense. ASF is asking PMC members to "test" on their platforms, but it's kinda vague what that means - I don't think we want to run a full precheckin!